### ISSUE TITLE
Version 2.2.0 introduced breaking changes (angular-oauth2-oidc)

### ISSUE THREAD

**Issue by unlight** [dankogai/js-base64#52]
This is bogus commit https://github.com/dankogai/js-base64/commit/dd715ce2fe8c96c8ddae0159e9eb2dfc0ce2d2c5

Part of source code of `angular-oauth2-oidc@1.0.20`
```
import {Base64} from 'js-base64';
var claimsJson = Base64.decode(claimsBase64);
```
```
var js_base64_1 = require('js-base64');
var claimsJson = js_base64_1.Base64.decode(claimsBase64); // js_base64_1.Base64 is undefined
```

Webpack converts to:
```
    // that's it!
    if (global['Meteor']) {
        Base64 = global.Base64; // for normal export in Meteor.js
    }
    if (typeof module !== 'undefined' && module.exports) {
        module.exports.Base64 = global.Base64;
    }
    if (true) {
        // AMD. Register as an anonymous module.
        !(__WEBPACK_AMD_DEFINE_ARRAY__ = [], __WEBPACK_AMD_DEFINE_RESULT__ = function(){ return global.Base64 }.apply(exports, __WEBPACK_AMD_DEFINE_ARRAY__),
				__WEBPACK_AMD_DEFINE_RESULT__ !== undefined && (module.exports = __WEBPACK_AMD_DEFINE_RESULT__));
    }
```

**Comment by unlight** [comment]
#46 
Please, consider to revert changes and bump version 3.0.0

**Comment by unlight** [comment]
Probably related #42

**Comment by unlight** [comment]
Workaround, disable AMD
```
module: {
  rules: [
    {
      parser: {
        amd: false,
      }
    },
}
...
```

**Comment by dankogai** [comment]
Aw, I just undid #46 and released it as 2.2.1.  I have to reconsider what to support and what not to…
